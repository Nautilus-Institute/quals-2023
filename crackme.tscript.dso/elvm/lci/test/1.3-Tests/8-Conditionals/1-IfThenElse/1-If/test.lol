HAI 1.3
	WIN
	O RLY?
		YA RLY
			VISIBLE "1a"
	OIC
	1
	O RLY?
		YA RLY
			VISIBLE "2a"
	OIC
	2.345
	O RLY?
		YA RLY
			VISIBLE "3a"
	OIC
	"abc"
	O RLY?
		YA RLY
			VISIBLE "4a"
	OIC
KTHXBYE
