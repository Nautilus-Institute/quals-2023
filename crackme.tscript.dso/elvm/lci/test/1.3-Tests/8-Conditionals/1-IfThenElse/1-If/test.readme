This test checks that the operation of the if branch in a simple if/then/else
block works correctly.
