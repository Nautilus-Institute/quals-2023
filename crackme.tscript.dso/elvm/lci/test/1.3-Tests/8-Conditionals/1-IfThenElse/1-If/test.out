1a
2a
3a
4a
